new/components/restservice/scripts/install_plugins.py

```diff
@@ -0,0 +1,78 @@
+import tarfile
+import tempfile
+import os
+from StringIO import StringIO
+
+import fabric
+import fabric.api
+import jinja2
+
+from cloudify import ctx
+
+
+def install_plugins():
+
+    root_dir = os.path.join(ctx._endpoint.storage.resources_root,
```

Comment:
no need for `ctx._endpoint.storage.resources_root`. When you `os.path.join`, and the n'th component is an absolute path (such as the case with `os.path.dirname(__file__)` the previous parts are imply discarded. So you can just do
`root_dir = os.path.dirname(__file__)`